webapp/src/Service/ConfigurationService.php

```diff
@@ -183,6 +185,152 @@ function (ConfigCacheInterface $cache) {
         return $specification;
     }
 
+    /**
+     * Save the changes from the given request
+     *
+     * @param Request         $request
+     * @param EventLogService $eventLog
+     * @param DOMJudgeService $dj
+     * @param string          $keyPrefix
+     *
+     * @throws NonUniqueResultException
+     */
+    public function saveChanges(
+        Request $request,
```

Comment:
Passing the `$request` to the the ConfigurationService feels a bit weird (although it's simple). But that means we can't really use it from a non-web context. Maybe better to pass the relevant information in a different structure?

Reply:
I now convert the request to an array and pass that. That seems to be more usable?